sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4115,6 +4115,7 @@ object SQLConf {
       .doc("Allow PruneFilters to remove streaming subplans when we encounter a false filter. " +
         "This flag is to restore prior buggy behavior for broken pipelines.")
       .version("4.0.0")
+      .withAlternative("spark.databricks.sql.optimizer.pruneFiltersCanPruneStreamingSubplan")
```

Comment:
@dongjoon-hyun 
Should we add this to DeprecatedConfig as well? Also, should we also file a JIRA ticket for removing?

Reply:
It's no harm to add it. When we remove the config in the future (throw an error if it's set), we need a ticket.